flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/harness/TemporalJoinHarnessTest.scala

```diff
@@ -92,6 +102,186 @@ class TemporalJoinHarnessTest extends HarnessTestBase {
 
   private val rexBuilder = new RexBuilder(typeFactory)
 
+  @Test
+  def testRowtime() {
+    val testHarness = createTestHarness(new OrdersRatesRowtimeTemporalJoinInfo())
+
+    testHarness.open()
+    val expectedOutput = new ConcurrentLinkedQueue[Object]()
+
+    // process without conversion rates
+    testHarness.processElement1(new StreamRecord(CRow(2L, "Euro", 0L), 0L))
```

Comment:
Introduce helper method to reduce duplicate code for `new StreamRecord(CRow(v1, v2, t), t)`

Reply:
I think this is resolved by the comment bellow?